rclpy_examples/package.xml

```diff
@@ -7,13 +7,15 @@
   <maintainer email="esteve@osrfoundation.org">Esteve Fernandez</maintainer>
   <license>Apache License 2.0</license>
 
+  <exec_depend>example_interfaces</exec_depend>
   <exec_depend>rclpy</exec_depend>
   <exec_depend>std_msgs</exec_depend>
 
   <test_depend>ament_copyright</test_depend>
   <test_depend>ament_pep257</test_depend>
   <test_depend>ament_pep8</test_depend>
   <test_depend>ament_pyflakes</test_depend>
+  <test_depend>example_interfaces</test_depend>
```

Comment:
I see the point. Is there any way to tell ament to construct the topological order taking the exec dependencies into account ? 
If I remove these test dependencies and run `ament.py test --end-with rclpy_examples` or `ament.py build --end-with rclpy_examples` this `rclpy_examples` package is higher than `rclpy` in the topological order so I can't run the generated executables because `rclpy` has never been compiled/installed